presentation/valid/ttml2-prstn-font-shear-linear-mapping.xml

```diff
@@ -11,46 +11,60 @@
       <ttm:desc>#fontShear - linear mapping</ttm:desc>
     </metadata>
     <styling>
-      <initial tts:color="white"/>
-      <initial tts:fontSize="1.000001c"/>
-      <style xml:id="s1" tts:showBackground="whenActive" tts:backgroundColor="black" tts:textAlign="center" tts:displayAlign="center"/>
+      <style xml:id="sRegion" tts:showBackground="whenActive" tts:backgroundColor="black" tts:displayAlign="center"/>
+      <style xml:id="sText" tts:color="white" tts:textAlign="center" tts:fontSize="6.67%" tts:lineHeight="8.0%"/>
```

Comment:
These new font and lineheight sizes are absurdly small, unless you switch the `%` units to `rh` units (which then makes the test rely also on support for that unit). The impact of small text is that it will make verifying the presentation results harder. It doesn't seem like the sizes are an important part of the test so why not leave them as defaults?

Reply:
See 0b48541.